common/persistence/dataInterfaces.go

```diff
@@ -2310,6 +2317,146 @@ func (e *WorkflowExecutionInfo) GetCurrentBranch() []byte {
 	return e.BranchToken
 }
 
+// UpdateWorkflowStateCloseStatus update the workflow state
+func (e *WorkflowExecutionInfo) UpdateWorkflowStateCloseStatus(
```

Comment:
consider moving this to a separate go file. datainterfaces.go was originally intended to just have the interfaces defined. This is more than a simple get/set method for this file.

Reply:
sure, will do in the next PR